lib/vm/src/instance.rs

```diff
@@ -296,13 +315,18 @@ impl Instance {
         match export {
             ExportIndex::Function(index) => {
                 let sig_index = &self.module.functions[*index];
-                let (address, vmctx) = if let Some(def_index) = self.module.local_func_index(*index)
-                {
-                    (self.functions[def_index].0 as *const _, self.vmctx_ptr())
-                } else {
-                    let import = self.imported_function(*index);
-                    (import.body, import.vmctx)
-                };
+                let (address, vmctx, function_ptr) =
```

Comment:
This needs to be revisited. I'm not sure why we need `function_ptr` in an exported function from the InstanceHandle